sample_factory_examples/dmlab/dmlab_params.py

```diff
@@ -4,20 +4,26 @@
 from sample_factory.utils.utils import str2bool
 
 
-def dmlab_override_defaults(env, parser):
+def dmlab_override_defaults(parser):
     parser.set_defaults(
         encoder_type="conv",
         encoder_subtype="convnet_impala",
         encoder_custom=None,
-        hidden_size=512,
+        hidden_size=256,
         obs_subtract_mean=0.0,
+        nonlinearity="relu",
         obs_scale=255.0,
         env_frameskip=4,
+        rollout=32,
+        recurrence=32,
+        rnn_type="lstm",
+        max_policy_lag=35,
+        num_epochs=1,
```

Comment:
Are these changes from the original IMPALA paper? Why do we decrease the hidden size?

Reply:
This is a comparable to the arguments you provide in the README. Should the example match the README, or would you like something else?